tasks/Qc.wdl

```diff
@@ -14,6 +14,33 @@ version 1.0
 ## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
 ## licensing information pertaining to the included programs.
 
+task LinkBamFile {
+  input {
+    File input_bam
+    String base_name
+    Boolean is_bam
+    Int preemptible_tries
+  }
+
+  Int disk_size = ceil(size(input_bam, "GiB")) + 20  
+
+  String bam_link = base_name + "." + if is_bam then "bam" else "cram"
```

Comment:
If you don't want to have an `is_bam` variable, you could maybe do something like this in the script:

```bash
INPUT_BAM="~{input_bam}"
EXTENSION="${INPUT_BAM##*.}"
ln -s "~{input_bam}" "~{base_name}.$EXTENSION"
```

Reply:
Is there a way I could do this outside of the command{} scope. Would this work there? I will need to define (or resolve) the `bam_link` so I can return it as an output of the Index task.

Reply:
The answer was no. only command{} is interpreted by the shell.